posthog/test/test_migration_0273.py

```diff
@@ -0,0 +1,175 @@
+import json
+from datetime import timedelta
+
+from django.utils import timezone
+from freezegun.api import freeze_time
+
+from posthog.test.base import TestMigrations
+
+
+@freeze_time("2021-08-25T13:00:00Z")
+class MarkInactiveExportsAsFinished(TestMigrations):
+    migrate_from = "0272_alter_organization_plugins_access_level"
+    migrate_to = "0273_mark_inactive_exports_as_finished"
+
+    def setUpBeforeMigration(self, apps):
+        Organization = apps.get_model("posthog", "Organization")
+        Team = apps.get_model("posthog", "Team")
+        Plugin = apps.get_model("posthog", "Plugin")
+        PluginConfig = apps.get_model("posthog", "PluginConfig")
+        PluginStorage = apps.get_model("posthog", "PluginStorage")
+
+        self.organization = Organization.objects.create()
+        self.team = Team.objects.create(organization=self.organization, app_urls=[])
+        self.plugins = [Plugin.objects.create(organization_id=self.organization.pk) for _ in range(6)]
+        self.plugin_configs = [
+            PluginConfig.objects.create(plugin=plugin, team=self.team, enabled=True, order=i)
+            for i, plugin in enumerate(self.plugins)
+        ]
+
+        # Case 1: Old non-finished export
+        self.create_entry(
+            apps,
+            self.plugin_configs[0].pk,
+            created_at=timezone.now() - timedelta(days=1),
+            activity="job_triggered",
+            detail={
+                "trigger": {
+                    "job_id": "1",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                }
+            },
+        )
+
+        # Case 2: Finished export
+        self.create_entry(
+            apps,
+            self.plugin_configs[1].pk,
+            created_at=timezone.now() - timedelta(days=1),
+            activity="job_triggered",
+            detail={
+                "trigger": {
+                    "job_id": "2",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                }
+            },
+        )
+        self.create_entry(
+            apps,
+            self.plugin_configs[1].pk,
+            created_at=timezone.now() - timedelta(days=1),
+            activity="export_success",
+            detail={
+                "trigger": {
+                    "job_id": "2",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                    "failure_reason": "Export was killed after too much inactivity",
```

Comment:
non-blocking

Probably not relevant but if this exported successfully i.e. `export_success` why would there be a failure_reason

Reply:
Copy-pasta fail - will fix!